src/views/NotFound.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div
+    class="flex items-center min-h-screen text-white bg-gradient-to-r from-blue-100 to-orange-100"
+  >
+    <div class="container flex flex-wrap items-center p-4 mx-auto">
+      <div class="w-full p-4 text-center md:w-5/12">
+        <img src="@/assets/images/404.svg" alt="Not Found" />
```

Comment:
Does this `src` works?

Reply:
Yes, but I'll change it into src="../assets/images/404.svg", is it okay?